src/phoenix/server/api/routers/v1/datasets.py

```diff
@@ -450,3 +516,100 @@ async def _parse_form_data(
         metadata_keys,
         file,
     )
+
+
+async def _get_dataset_download_csv(request: Request) -> Response:
+    encoded_dataset_id = request.path_params.get("datasetId")
+    if not encoded_dataset_id:
+        return Response(
+            content="Missing Dataset ID",
+            status_code=HTTP_422_UNPROCESSABLE_ENTITY,
+        )
+    dataset_global_id = GlobalID.from_id(encoded_dataset_id)
+    if dataset_global_id.type_name != Dataset.__name__:
+        return Response(
+            content=f"Invalid Dataset ID: {encoded_dataset_id}",
```

Comment:
Suggestion: Explain why the dataset ID is invalid, i.e., because it encodes an incorrect type name, but we expect "Dataset".

Reply:
I think the ID is supposed to be opaque, so we don't want to be tied down to a specific implementation

Reply:
Curious, do we need to use the relay IDs here? Any reason to not just use the rowid?

Reply:
I guess it makes it simpler if someone wants to go to the GraphQL API.